src/custom/pages/Rewards/index.tsx

```diff
@@ -2,36 +2,29 @@ import React from 'react'
 import Page, { Title, Content, GdocsListStyle } from 'components/Page'
 import styled from 'styled-components'
 import { Trans } from '@lingui/macro'
-import CowsImg from 'assets/images/cows.png'
+import CowsImg from 'custom/assets/cow-swap/cows-side-by-side.png'
+import Column from 'components/Column'
```

Comment:
Row and Column components aren't used. Maybe we can delete them

Reply:
It's actually a must :)

The build is failing because of that

![screenshot_2021-07-17_10-32-36](https://user-images.githubusercontent.com/43217/126045223-2c0f9b36-e820-4ab8-8a46-5d75caa3a099.png)